html/modules/custom/bc_dc/bc_dc.module

```diff
@@ -276,3 +276,22 @@ function bc_dc_cron(): void {
   $reviewReminder = \Drupal::service('bc_dc.review_reminder');
   $reviewReminder->sendRemindersToAllUsers();
 }
+
+/**
+ * Implements hook_form_alter.
+ */
+function bc_dc_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $form_state, $form_id) {
+  if ($form['form_id']['#id'] === 'edit-bc-dc-add-columns-form') {
+    $nid = $form_state->getStorage('nid');
+    $paragraph = \Drupal::entityTypeManager()->getStorage('node')->load($nid['nid']);
```

Comment:
Check that the node is the correct bundle.
`$paragraph` should be `$entity`.

Reply:
fixed